zap/middleware.go

```diff
@@ -38,22 +38,17 @@ func (w *Logger) EchoMiddleware(l logi.WLogger) echo.MiddlewareFunc {
 
 			fields := []any{
 				"remote_ip", c.RealIP(),
+				key.External, false,
 				key.Email, email,
 				key.CompanyKey, companyKey,
 				key.Latency, time.Since(start).Milliseconds(),
 				key.Method, req.Method,
 				key.URI, req.RequestURI,
+				key.Path, req.URL.Path,
```

Comment:
In some cases, I cannot extract PATH from `URI` in datadog. This will be very helpful